deposit/src/main/java/edu/unc/lib/deposit/fcrepo3/MakeFOXML.java

```diff
@@ -343,14 +343,19 @@ private void writeDepositRecord(Resource deposit) {
 		}
 
 		// add manifest DS
-		File manifest = getManifestFile();
-		log.info("Adding manifest file to foxml {}", manifest);
-		if (manifest != null && manifest.exists()) {
-			String dsLabel = Datastream.DATA_MANIFEST.getLabel();
-			Element el = FOXMLJDOMUtil.makeLocatorDatastream(Datastream.DATA_MANIFEST.getName(),
-					"M", manifest.toURI().toString(), "text/xml", "URL", dsLabel, false, null);
-			foxml.getRootElement().addContent(el);
-			log.info("Manifest file exists and has been added");
+		String dsLabel = Datastream.DATA_MANIFEST.getLabel();
+		List<String> filePaths = getManifestFileURIs();
+		if (!filePaths.isEmpty()) {
+			int i = 0;
+			Element el ;
+			for (String path : filePaths) {
+				el	= FOXMLJDOMUtil.makeLocatorDatastream(Datastream.DATA_MANIFEST.getName() + i++,
+						"M", path, "text/xml", "URL", dsLabel, false, null);
+				foxml.getRootElement().addContent(el);
+				log.info("Manifest files have been added to foxml");
```

Comment:
This log statement should either be moved out of the for loop with the count of the number of manifest files added, or it should read "Manifest file {} has been added to foxml", with the name of the manifest file included there.  The syntax for that is 
log.info("message {}", insertValue)